Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/client/ClientPortal.tsx

```diff
@@ -90,21 +83,25 @@ const ClientPortal = () => {
             Element: ContestRegister,
         },
         {
-            path: '/contest/problems/:contestId',
+            path: '/contests/problems/:contestId',
             Element: ContestProblemsPage,
         },
         {
-            path: '/contest/edit/:contestId',
+            path: '/contests/edit/:contestId',
             Element: ContestEditPage,
         },
         {
             path: '/contests/:contestId/:participationType',
             Element: ContestSolutionSubmitPage,
         },
         {
-            path: '/contests/:contestId/:participationType/results/:resultType',
+            path: '/contest/:contestId/:participationType/results/:resultType',
```

Comment:
Are we sure it is `contest` here, and not `contests`?

Reply:
nice catch, fixed